### PR TITLE
Dev spell

### PR DESCRIPTION
**Pull request by RlonRyan**
Implement Spell Registry.

### CODE REVIEW

File: src/main/java/com/teaminfinity/elementalinvocations/impl/spell/SpellRegistry.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	}
+	
+	@Override
+	public List<Spell> getSpells() {
+		return new ArrayList<>(spells.values());
+	}
+	
+	@Override
+	public List<Spell> getSpells(Element... elements) {
+		return getSpells(Arrays.asList(elements));
```
Comment (InfinityRaider) [inline src/main/java/com/teaminfinity/elementalinvocations/impl/spell/SpellRegistry.java]: a spell should be combination unique.

Comment (RlonRyan) [inline src/main/java/com/teaminfinity/elementalinvocations/impl/spell/SpellRegistry.java]: Ordered?

Comment (InfinityRaider) [inline src/main/java/com/teaminfinity/elementalinvocations/impl/spell/SpellRegistry.java]: No, just match the number of charges.

Comment (RlonRyan) [inline src/main/java/com/teaminfinity/elementalinvocations/impl/spell/SpellRegistry.java]: That will take a bit. Let me commit these changes first.

**Pull request merged**